app/modules/asset_groups/metadata.py

```diff
@@ -0,0 +1,320 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=bad-continuation
+"""
+Asset_groups metadata
+Classes for parsing and validating the received metadata in requests
+--------------------------
+"""
+import enum
+import logging
+import os
+
+from flask import current_app
+from flask_login import current_user  # NOQA
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class AssetGroupMetadataError(Exception):
```

Comment:
I like this approach to errs, if we follow the pattern it should make hooking in an AuditLog easier.

Reply:
Good idea. 
This should probably (in a separate PR) move to somewhere more generic. There's nothing in this Exception that's specific to AssetGroupMetadata.